lib/queue.js

```diff
@@ -184,7 +191,10 @@ class Queue extends Emitter {
     const cleanup = () => {
       this._isClosed = true;
 
-      const clients = [this.client];
+      const clients = [];
+      if (this.settings.quitCommandClient) {
```

Comment:
semi-uninformed question, I think it's fine as is and I probably just need reminding: on line 204 we do `client.quit()` with each of these; should we keep doing that or would we want to use new `disconnect()` helper? I don't recall exact differences of `end()` vs `quit()` and why we use which one where.